gradle/publishable-java-library.gradle

```diff
@@ -0,0 +1,46 @@
+def licenseFiles = copySpec {
+    //mockito license
+    from(".") { include 'LICENSE' }
+}
+
+//TODO why do we still ship the main jar with sources?
```

Comment:
Yup, it's an old issue: #158. What was the original reason to do that? Is it still present?